package/wpe/wpewebkit/wpewebkit.mk

```diff
@@ -194,8 +194,12 @@ endif
 
 endif
 
-ifeq ($(BR2_PACKAGE_WPEWEBKIT_DEBUG)$(BR2_PACKAGE_WPEWEBKIT_SYMBOLS),y)
-WPEWEBKIT_SYMBOL_FLAGS = -g
+ifeq ($(BR2_PACKAGE_WPEWEBKIT_DEBUG)$(BR2_PACKAGE_WPEWEBKIT_SYMBOLS_LEVEL_2),y)
+WPEWEBKIT_SYMBOL_FLAGS = -g2
+else ifeq ($(BR2_PACKAGE_WPEWEBKIT_SYMBOLS_LEVEL_1),y)
+WPEWEBKIT_SYMBOL_FLAGS = -g1
+else ifeq ($(BR2_PACKAGE_WPEWEBKIT_SYMBOLS_LEVEL_3),y)
+WPEWEBKIT_SYMBOL_FLAGS = -g3
```

Comment:
This is missing a case for `BR2_PACKAGE_WPEWEBKIT_SYMBOLS_NO`, which would map to `-g0`.

Reply:
But isn't this equivalent to not adding the -g stuff? That's what it is doing, right? Anyway, not big deal of a change.

Reply:
The reason why I suggested removing the check on `BR2_ENABLE_DEBUG` and handling the `-g0` case is to allow more flexibility.

If a debug build is configured globally with `BR2_ENABLE_DEBUG`, then Buildroot will add `-g` to the compiler and linker flags. Always setting a value for `WPEWEBKIT_SYMBOL_FLAGS` would allow passing `-g0` _after_ the flag added by Buildroot (resulting in `-g ... -g0`, the second flag takes precedence), resulting in the whole system having debug symbols, WebKit built in debug mode (with assertions on), but without debuginfo&mdash;which is a configuration people might want sometimes.